src/gf-platformid

```diff
@@ -47,15 +47,20 @@ coreos_gf download "${blscfg_path}" "${tmpd}"/bls.conf
 # Remove any platformid currently there
 sed -i -e 's, ignition.platform.id=[a-zA-Z0-9]*,,g' "${tmpd}"/bls.conf
 sed -i -e 's,^\(options .*\),\1 ignition.platform.id='"${platformid}"',' "${tmpd}"/bls.conf
-coreos_gf upload "${tmpd}"/bls.conf "${blscfg_path}"
 
 if [ "$basearch" = "s390x" ] ; then
+    # Before we re-run zipl make sure we have the firstboot options
```

Comment:
Probably also worth mentioning that this is a hack and cross-referencing `create_disk.sh` (and adding a reference to this code too there) so we keep track of where this hack lives.

Reply:
yes.will do.